lib/mail_room/connection.rb

```diff
@@ -27,7 +27,7 @@ def idling?
     # is the imap connection closed?
     # @return [Boolean]
     def disconnected?
-      @imap.disconnected?
+      imap.disconnected?
```

Comment:
we could also use `@imap&.disconnected` here, which will also be falsey, if the instance variable needs to be used.